resource/js/components/PageHistory.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+
+import Icon from './Common/Icon';
+import PageRevisionList from './PageHistory/PageRevisionList';
+
+export default class PageHistory extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.crowi = window.crowi; // FIXME
```

Comment:
How about changing `crowi` to `props`?
see: https://github.com/crowi/crowi/pull/174/files#diff-39992ee08af86834f2b5095ded60c098R37